frontend/src/views/category/Category.vue

```diff
@@ -16,14 +16,24 @@
 
       <template #title-actions>
         <TogglePaperSize v-model="isPaperDisplaySize" />
-        <v-menu v-if="isManager" offset-y>
+        <v-menu offset-y>
```

Comment:
Ah, like this you can copy categories from camps where you are not a manager, nice.

Reply:
Exactly. Same as for the activities which we already allow to copy. Sharing is caring.